src/notation/view/pianokeyboard/pianokeyboardpanelcontextmenumodel.cpp

```diff
@@ -167,6 +203,35 @@ MenuItem* PianoKeyboardPanelContextMenuModel::makeNumberOfKeysItem(const muse::T
     return item;
 }
 
+MenuItem* PianoKeyboardPanelContextMenuModel::makeToggleNotatedPitchItem(const muse::TranslatableString& title, bool state)
```

Comment:
More meaningful parameter name, to prevent confusion:
```suggestion
MenuItem* PianoKeyboardPanelContextMenuModel::makeToggleNotatedPitchItem(const muse::TranslatableString& title, bool isWrittenPitch)
```